src/endpoints/index.ts

```diff
@@ -3,5 +3,6 @@ import { Request, Response } from 'express';
 export type Endpoint = () => (req: Request, res: Response) => void;
 
 export const HealthCheck: Endpoint = () => (_req: Request, res: Response): void => {
+    console.log('Im up and running!!!!!!');
```

Comment:
Was this intentionally left in? If so not a problem, but just wanted to double check!

Reply:
No it wasn't meant to be included! thanks for the spot